controllers/storagecluster/storageclasses.go

```diff
@@ -280,6 +281,45 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	}
 }
 
+// newCephBlockPoolVirtualizationStorageClassConfiguration generates configuration options for a Ceph Block Pool StorageClass for virtualization environment.
+func newCephBlockPoolVirtualizationStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
```

Comment:
the content of this function is exact copy of `newCephBlockPoolStorageClassConfiguration` can this function call `newCephBlockPoolStorageClassConfiguration` and update the StorageClassConfiguration.storageClass.Parameters to add the new map option?

Reply:
Right, We are changing just a different Name, Annotation & 2 extra Parameters. So this seems like a good way for simplification.